### DISCUSSION TITLE
Improved graphics

### DISCUSSION THREAD

**Discussion by JZeravik** [Keriew/augustus#970] [Ideas]
Hi, would be possible to upscale all assets (pictures), to higher resolution? Upscale and improve quality should be easy nowadays using AI. But could it work? Would it be complicated?

**Comment by crudelios** [comment]
Hi!

Our issue isn't technical at all, but rather of copyright.

We can't distribute the original assets because we would be violating the game's copyright. That's why an original installation is required for the game to run.

Because of that, we can't also provide rescaled/improved versions of the original assets.

**Reply by JZeravik** [reply]
That's really disappointing, sadly I have original game  had to buy it. Just maybe few questions is there any way to unpack original assets when I have steam version of game? I have seen some unpacker but it unpack it from an installer. When I look on assets for Augustus, in case I would want to rescale them just for my personal use, is enought to rescale the pictures to 4x resolution and then basically multiply by factor 4 all coordinates and sizes in corresponding *.XML files? Or there could be some catch?

**Reply by crudelios** [reply]
Yeah there would be a catch. There's no way right now to scale assets, so if you made them 4x bigger they would also appear 4x as large (and not scale and look crisper).

Also, original assets are all loaded in a different way than the custom assets, so you'd need to replace the code loading from the original assets to use the custom ones.

Both those issues could be fixed by editing the code, but that requires delving into the game's code and then properly debugged, etc.

Regarding the unpacker, you can use [sgreader](https://github.com/bvschaik/citybuilding-tools/releases/tag/sgreader-v1.1), you just need to point the program to the files inside the C3 folder that Steam creates (usually inside `steamapps`).
